assembl/static2/js/app/components/debate/survey/post.jsx

```diff
@@ -51,31 +53,35 @@ class Post extends React.Component {
   }
   handleAddSentiment(target, type) {
     const { id } = this.props;
-    this.props.addSentiment({ variables: { postId: id, type: type } })
-    .then((sentiments) => {
-      target.setAttribute('class', 'sentiment sentiment-active');
-      this.setState({
-        like: sentiments.data.addSentiment.post.sentimentCounts.like,
-        disagree: sentiments.data.addSentiment.post.sentimentCounts.disagree,
-        mySentiment: sentiments.data.addSentiment.post.mySentiment
+    this.props
+      .addSentiment({ variables: { postId: id, type: type } })
+      .then((sentiments) => {
+        target.setAttribute('class', 'sentiment sentiment-active');
+        this.setState({
+          like: sentiments.data.addSentiment.post.sentimentCounts.like,
+          disagree: sentiments.data.addSentiment.post.sentimentCounts.disagree,
+          mySentiment: sentiments.data.addSentiment.post.mySentiment
+        });
+      })
+      .catch((error) => {
+        displayAlert('danger', `${error}`);
       });
-    }).catch((error) => {
-      displayAlert('danger', `${error}`);
-    });
   }
   handleDeleteSentiment(target) {
     const { id } = this.props;
-    this.props.deleteSentiment({ variables: { postId: id } })
-    .then((sentiments) => {
-      target.setAttribute('class', 'sentiment');
-      this.setState({
-        like: sentiments.data.deleteSentiment.post.sentimentCounts.like,
-        disagree: sentiments.data.deleteSentiment.post.sentimentCounts.disagree,
-        mySentiment: sentiments.data.deleteSentiment.post.mySentiment
+    this.props
+      .deleteSentiment({ variables: { postId: id } })
+      .then((sentiments) => {
+        target.setAttribute('class', 'sentiment');
+        this.setState({
+          like: sentiments.data.deleteSentiment.post.sentimentCounts.like,
+          disagree: sentiments.data.deleteSentiment.post.sentimentCounts.disagree,
+          mySentiment: sentiments.data.deleteSentiment.post.mySentiment
```

Comment:
Yes, we should avoid the use of the redux store to store data that comes from apollo when we don't need to. 
Here we probably just need a `post` variable to lighten the code. We should also stay aware of code that might be use lots of time and refactor as soon as we are about to duplicate some lines of code.